less/mosaico/static-pages/abi.less

```diff
@@ -0,0 +1,16 @@
+.static-page__abi {
```

Comment:
The page overflows on mobile. There is a static-page.less file that can be used, especially if the font should be as the other static pages (using Duplex sans for headlines and Roboto for the rest of the things).